src/axom/sidre/interface/sidre_shroud.yaml

```diff
@@ -659,9 +688,11 @@ declarations:
    # The concept of a valid name is different for C++, C and Fortran
   options:
     F_string_len_trim: false
-  format:
-    C_code:  return name != NULL;
-    F_code:  '{F_result} = name .ne. " "'
+  splicer:
+    c:
+    - "return name != NULL;"
+    f:
+    - 'SHT_rv = name .ne. " "'
```

Comment:
@ltaylor16 -- could you please clarify this binding?

Does this completely replace the function call with the given splicer code, or is this added to the function call?

Reply:
It will completely replace the generated code.